src/analyzer.js

```diff
@@ -138,32 +158,38 @@ Analyzer.computeColMeta = function computeColMeta(
     if (!type) {
       type = allValidators.find(buildValidatorFinder(data, columnName));
     }
-    // if theres still no type, dump this column
     var category = Analyzer._category(type);
+    var colMeta = {
```

Comment:
Not super happy with this name, but it was the best I could come up with - suggestions welcome 😄